manifests/overlays/prod/applications/data_hub/dh-stage-rsyslog.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Can you add this to the kustomization file [here](https://github.com/AICoE/aicoe-cd/blob/master/manifests/overlays/prod/applications/data_hub/kustomization.yaml):

Reply:
Done. I don't think that step is in the docs anywhere (if it is I missed it). I'm not sure where to add it.

Reply:
hrmm it's [here](https://github.com/AICoE/aicoe-cd/blob/master/docs/create_argocd_application_manifest.md#step-1-create-the-application-yaml) at the bottom of step 1

Reply:
ah ok, I was following the instructions [here](https://github.com/AICoE/aicoe-cd/blob/master/docs/get_argocd_to_manage_your_app.md)